pkg/netns/netns_linux.go

```diff
@@ -52,7 +52,15 @@ import (
 // path to the network namespace.
 func newNS(baseDir string) (nsPath string, err error) {
 	b := make([]byte, 16)
-	if _, err := rand.Reader.Read(b); err != nil {
+
+	n, err := rand.Reader.Read(b)
```

Comment:
what about just use `rand.Read()` with ReadFull() ?

https://pkg.go.dev/crypto/rand#Read

Reply:
done..

Reply:
note added a lock "just in case" after discussing with @MikeZappa87 last night :-)